### DISCUSSION TITLE
A Slightly Different Post

### DISCUSSION THREAD

**Discussion by pguli0** [pocketbase/pocketbase#2130] [Show and tell]
It's incredible to see what PocketBase has achieved. In fact, it is so massive that people from VCs started to lay their eyes on it too :) 

Feel free to check us out at https://picuscap.com and reach out - Always looking for cutting-edge tech and to help communities around it.

**Comment by ganigeorgiev** [comment]
Please don't spam the discussions like this. I've received your 2 emails and silently ignored them because I got tired replying the same to every investor that contacted me so far.

I'm not looking for creating a startup nor have any monetization plans for PocketBase at this stage.

If you want to support the PocketBase development, feel free to contribute through our [Open Collective page](https://opencollective.com/pocketbase).

**Comment by pguli0** [comment]
Got it, a reply always makes things clear and easier for everyone.

Happy coding
